linker/shared/src/main/scala/org/scalajs/linker/checker/ClassDefChecker.scala

```diff
@@ -1050,6 +1051,10 @@ private final class ClassDefChecker(classDef: ClassDef,
       case CreateJSClass(className, captureValues) =>
         checkTrees(captureValues, env)
 
+      case Transient(Transients.PackLong(lo, hi)) if featureSet.supports(FeatureSet.PackLongTransient) =>
```

Comment:
Similar issue then in IR checker (just misleading, not incorrect here).

https://github.com/scala-js/scala-js/pull/5205/files#r2586435012